### PR TITLE
Add status about dnf5 in Rawhide

### PR DESCRIPTION
**Pull request by lachmanfrantisek**

### CODE REVIEW

**Review by majamassarini — APPROVED** [review]

File: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+date: "2023-06-22T09:30:00+02:00"
+affected:
+  - Testing Farm
+resolved: false
+section: issue
+severity: disrupted
+---
+
+The dnf5 came to rawhide resulting in various of issues
+when running test in Testing Farm on Rawhide.
```
Comment (nforro) [inline content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:12]:
```suggestion
dnf has been upgraded to dnf5 in Rawhide, resulting in various issues
when running tests in Testing Farm.
```

File: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The Testing Farm team is fixing the issues
+and trying to resolve the issues soon.
```
Comment (nforro) [inline content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:15]:
```suggestion
The Testing Farm team is working on fixing the issues
and trying to resolve the situation ASAP.
```

File: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+severity: disrupted
+---
+
+dnf has been upgraded to dnf5 in Rawhide, resulting in various issues
+when running tests in Testing Farm.
+
+The Testing Farm team is working on fixing the issues
+and trying to resolve the situation ASAP.
+
+If you want to help, here are the issues in for DNF5 itself:
```
Comment (nforro) [inline content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:17]:
```suggestion
If you want to help, here are upstream issues related to this problem:
```

File: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Fedora Rawhide Testing Blocked on Dnf5"
```
Comment (nforro) [inline content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:2]: I think we should either use dnf/dnf5, or DNF/DNF5.

```suggestion
title: "Fedora Rawhide Testing Blocked on dnf5"
```

File: content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+dnf has been upgraded to dnf5 in Rawhide, resulting in various issues
+when running tests in Testing Farm.
+
+The Testing Farm team is working on fixing the issues
+and trying to resolve the situation ASAP.
+
+If you want to help, here are the issues in for DNF5 itself:
+
+- https://github.com/rpm-software-management/dnf5/issues/549
+- https://github.com/rpm-software-management/dnf5/issues/168
```
Comment (lbarcziova) [inline content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:20]:
```suggestion
The Testing Farm team together with dnf5 team is working on fixing the issues
and trying to resolve the situation ASAP.

Related DNF5 issues:

- https://github.com/rpm-software-management/dnf5/issues/549
- https://github.com/rpm-software-management/dnf5/issues/168
```

Comment (lachmanfrantisek) [reply content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md:20]: I wanted to suggest people to contribute..;-)

Comment (nforro) [reply content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md]: @lachmanfrantisek Could you also include the dnf5 team part from Laura's suggestion?

Comment (nforro) [reply content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md]: Ah, never mind :slightly_smiling_face:

Comment (lachmanfrantisek) [reply content/issues/2023-06-22-fedora-rawhide-testing-blocked-on-dnf5.md]: I've included the first suggestion, but let the other as is if you don't mind.

**Review by nforro — APPROVED** [review]

**Comment by lachmanfrantisek** [comment]
Thanks for all the suggestions!

**Pull request merged**